parsec/api/protocol/invite.py

```diff
@@ -1,13 +1,15 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
 
 from enum import Enum
-from typing import Dict, cast, TYPE_CHECKING
+from typing import Dict, cast
 
-from parsec.types import UUID4
 from parsec.serde import BaseSchema, OneOfSchema, fields
 from parsec.api.protocol.base import BaseReqSchema, BaseRepSchema, CmdSerializer
 from parsec.api.protocol.types import HumanHandleField, UserIDField
 
+from parsec._parsec import (
```

Comment:
Why parenthesis ?

Reply:
Missing `black` formatting

Reply:
done, but I've to manually format this line